docs/sources/panels-visualizations/visualizations/xy-chart/index.md

```diff
@@ -0,0 +1,230 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/xy-chart/
+keywords:
+  - grafana
+  - chart
+  - xy chart
+  - documentation
+  - guide
+  - graph
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+description: Configure options for Grafana's xy chart
+title: XY chart
+weight: 100
+---
+
+# XY chart
+
+XY charts provide a way to visualize...
+
+<!--add brief description here-->
+
+![An xy chart showing height weight distribution](screenshot-xy-chart-v10.4.png)
+
+<!--add more ideal use cases/examples here-->
+
+## Supported data formats
+
+TBA
+
+## XY chart options
+
+### Series mapping
+
+Set how series data is mapped in the visualization.
+
+<!--i don't understand these descriptions-->
+
+- **Table** - Plot values within a single table result.
+- **Manual** - Construct values from any result. You can add multiple series for which you'll set the **X-field** and **Y-field** options. When you choose this option, the **Point color** and **Point size** fields are also displayed.
+
+### Data
+
+<!--what is the scenario where you need to choose a data set?-->
+
+Select your data set. This option is only available when you set **Series mapping** to **Table** mode.
+
+### X-field
+
+Select which field the x-axis represents.
+
+<!--can we make clear here how this is different when you map your series by table or manual; there seem to be more fields to choose from on manual mode-->
+
+### Y-fields
+
+Select which fields the y-axis represents.
+
+**Table** mode - When you set the series mapping to **Table** mode, you can only update the x-field and the y-fields are automatically assigned based on that selection. In this mode you can enable and disable the **Y-field**.
```

Comment:
```suggestion
**Table** mode - When you set the series mapping to **Table** mode, you can only update the x-field and the y-fields are automatically assigned based on that selection. In this mode you can enable and disable the **Y-field**.

For example, if your data source is a table with five columns, you may want to disable one or more of the columns so that they're not included in the visualization.
```